hudi-integ-test/pom.xml

```diff
@@ -525,24 +519,9 @@
         </executions>
       </plugin>
       <plugin>
-      <groupId>org.scalatest</groupId>
```

Comment:
Why deleting this config?

Reply:
I moved the configuration up to the parent pom so we didn't need to define the configuration in ever module that wanted to use the plugin. This allows us to also specify the logging configuration once instead of hoping each module sets it properly.